storybook/src/pages/amsterdam/ProductPage/ProductPage.tsx

```diff
@@ -0,0 +1,133 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+import {
+  Breadcrumb,
+  CallToActionLink,
+  Grid,
+  Heading,
+  Link,
+  Paragraph,
+  UnorderedList,
+} from '@amsterdam/design-system-react'
+
+export const ProductPage = () => (
+  <Grid paddingBottom="x-large" paddingTop="large">
+    <Grid.Cell span={{ narrow: 4, medium: 5, wide: 7 }} start={{ narrow: 1, medium: 2, wide: 3 }}>
+      <Breadcrumb>
+        <Breadcrumb.Link>Home</Breadcrumb.Link>
+        <Breadcrumb.Link>Stadspas</Breadcrumb.Link>
+      </Breadcrumb>
+      <Heading className="ams-mb-xl" level={1}>
+        Gratis laptop of tablet voor de basisschool aanvragen
+      </Heading>
+      <Paragraph className="ams-mb-2xl" size="large">
+        U krijgt per huishouden 1 keer per 5 schooljaren een gratis laptop of tablet op de basisschool.
+      </Paragraph>
+
+      <Heading className="ams-mb-xs" level={2}>
```

Comment:
We currently use [ams-mb-s](https://github.com/Amsterdam/design-system/blob/develop/storybook/src/pages/amsterdam/HomePage/HomeSpotlight.tsx#L12), [ams-mb-m](https://github.com/Amsterdam/design-system/blob/develop/storybook/src/pages/amsterdam/ArticlePage/ArticleBody.tsx#L34) and now `ams-mb-xs` as the space between a level 2 Heading and the rest of the text. Those should all be the same, right?

`ams-mb-xs` feels a little cramped imo.